google/golang_utils.rb

```diff
@@ -29,6 +29,8 @@ def go_literal(value)
         "\"#{value}\""
       elsif value.is_a?(Numeric)
         value.to_s
+      elsif value.is_a?(Symbol)
+        "\"#{value}\""
```

Comment:
This is petty but if you want to treat symbols and strings the same I'd rather see

`if value.is_a?(String) or value.is_a?(Symbol)`

Reply:
Good idea. Done.